sandbox/apparmor/apparmor.go

```diff
@@ -663,7 +663,7 @@ func probeKernelFeatures() ([]string, error) {
 			features = append(features, "policy:permstable32:"+feat)
 		}
 	}
-	if data, err := os.ReadFile(filepath.Join(rootPath, featuresSysPath, "policy", "notify", "user")); err != nil {
+	if data, err := os.ReadFile(filepath.Join(rootPath, featuresSysPath, "policy", "notify", "user")); err == nil {
```

Comment:
heh, it's a file indeed:
```
root@uv2:~# cat /sys/kernel/security/apparmor/features/policy/notify/user 
file
```
we have a unit tests which uses `policy:notify:user:file` but it mocks the detected kernel features list rather than files under mocked /sys/kernel/security/features, so clearly this branch wasn't covered